src/session.rs

```diff
@@ -108,9 +142,17 @@ impl Session {
             // see https://github.com/httpie/httpie/commit/e09b74021c9c955fd7c3bab11f22801aaf9dc1b8
             // we will also ignore cookies as they are taken care of by save_cookies()
             if key != "cookie" && !key.starts_with("content-") && !key.starts_with("if-") {
-                self.content
-                    .headers
-                    .insert(key.into(), value.to_str()?.into());
+                match self.content.headers {
+                    Headers::Map(ref mut headers) => {
+                        headers.insert(key.into(), value.to_str()?.into());
+                    }
+                    Headers::List(ref mut headers) => {
+                        headers.push(Header {
```

Comment:
**Note to self**: overwrite old entries to avoid accumulation of headers

Reply:
Do you mean clearing the container before filling it or something else?

Reply:
Yeah, I meant clearing the container. So right now, If I make multiple requests with different `Authorization` headers, old values are not being removed

```json
{
    "__meta__": {
        "about": "xh session file",
        "xh": "0.18.0"
    },
    "auth": {
        "type": null,
        "raw_auth": null
    },
    "cookies": {},
    "headers": [
        {
            "key": "authorization",
            "value": "bearer secret-1"
        },
        {
            "key": "authorization",
            "value": "bearer secret-2"
        },
        {
            "key": "hello",
            "value": "world"
        }
    ]
}
```